content/configuration/node.md

```diff
@@ -109,9 +120,18 @@ Default: `true`
 
 `boolean | "mock" | "empty"`
 
-Many other Node.js core libraries can be configured as well. See the list of [Node.js core libraries and their polyfills](https://github.com/webpack/node-libs-browser).
+W> This option is only activated (via `NodeSourcePlugin`) when the target is unspecified, "browser" or "webworker".
+
+Polyfills for Node.js core libraries from
+[`node-libs-browser`](https://github.com/webpack/node-libs-browser) are used if
+available, when the `NodeSourcePlugin` plugin is enabled.
+See the list of [Node.js core libraries and their polyfills](https://github.com/webpack/node-libs-browser#readme).
+
+By default, webpack will polyfill each library if there is a known polyfill or do nothing if there is not one.
+In the latter case, webpack will behave as if the module name was configured with the `false` value.
 
-By default, Webpack will polyfill each library if there is a known polyfill or do nothing if there is not one.
+T> To import a built-in module, use [`__non_webpack_require__`](/api/module-variables/#__non_webpack_require__-webpack-specific-),
```

Comment:
Also. Importing using the default require instead of a polyfill, mock, or erroring.